From 972688cd968c187969950f3f278534d89c62b9cd Mon Sep 17 00:00:00 2001
From: Dave Gerlach <d-gerlach@ti.com>
Date: Tue, 13 Dec 2016 14:36:11 -0600
Subject: [PATCH 013/205] sched / idle: Export cpu_idle_poll_ctrl

This commit comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

Export cpu_idle_poll_ctrl so that it can be used in modules.

Signed-off-by: Dave Gerlach <d-gerlach@ti.com>
(cherry picked from commit 1bd61b7408a70ac537bb2c6a9e502b611d5f3ae2)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 kernel/sched/idle.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/sched/idle.c b/kernel/sched/idle.c
index ef63adc..0c3a7eb 100644
--- a/kernel/sched/idle.c
+++ b/kernel/sched/idle.c
@@ -41,6 +41,7 @@ void cpu_idle_poll_ctrl(bool enable)
 		WARN_ON_ONCE(cpu_idle_force_poll < 0);
 	}
 }
+EXPORT_SYMBOL(cpu_idle_poll_ctrl);
 
 #ifdef CONFIG_GENERIC_IDLE_POLL_SETUP
 static int __init cpu_idle_poll_setup(char *__unused)
-- 
1.7.9.5

